### PR TITLE
Willys/guidance/planning1

### PR DESCRIPTION
**Pull request by wpschaub**
Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Guidance/blob/master/.github/CLA.md).

Changes proposed in this pull request:  
- First DRAFT of the planning article, converted from dated guidance.
- Included the "dated" capacity workbook, which needs to be updated.

@ALM-Rangers/guidance

### CODE REVIEW

File: src/Planning/understand-options-tfs-vsts-environments.md [hunk 1]
```diff
@@ -1,33 +1,94 @@
...
+
+![Evolution from datacenters to cloud](_img/understand-options-tfs-vsts-environments/understand-options-tfs-vsts-environments-saas.png)
+
+While both TFS and VSTS provide the same essential services, VSTS provides your organization these added benefits:
+
+- **Always Latest** - No more expensive and complex upgrade cycles to manage your data center. You always have access to the latest version of the service, with optional access to preview features.
+- **CAPEX -> OPEX** - No servers and infrastructure to support, manage, and upgrade. You can move your organizations from resource optimized model, based on capital expenses (CAPEX), to a market optimised business model, based on operational expenses (OPEX). Read [how Microsoft builds its fast and reliable global network](https://azure.microsoft.com/en-gb/blog/how-microsoft-builds-its-fast-and-reliable-global-network/) to understand how Microsoft is continuously innovating and investing their cloud.
```
Comment (kenmuse) [inline src/Planning/understand-options-tfs-vsts-environments.md]: Should the link on building the fast and reliable network be part of the **Scalability** topic? The content of the linked article is more closely related to that topic.

File: src/Planning/understand-options-tfs-vsts-environments.md [hunk 2]
```diff
@@ -1,33 +1,94 @@
...
+The following table is a quick summary of the three deployment options, with suggested maximum number of conservative users. For a more detailed breakdown of the scenarios and recommended hardware, use the capacity planning workbook from the [TFS Planning, Disaster Avoidance and Recovery](https://vsardata.blob.core.windows.net/projects/Rangers_vsarPlanninglGuide_Everything.zip) guidance.
+
+|Deployment Option|Requests per second (RPS)|Max Users|
+|-----------------|--:|--------:|
+|Single Server|92-180|250-500|
+|Dual Servers (scale up)|476-730|2200-3600|
+|Multi Servers (scale out)|730+|3600+|
+
+>
+> [!TIP]
```
Comment (kenmuse) [inline src/Planning/understand-options-tfs-vsts-environments.md]: Is it worth mentioning that the deployment option can be changed after the fact (or that you can migrate to VSTS)?

File: src/Planning/understand-options-tfs-vsts-environments.md [hunk 3]
```diff
@@ -1,33 +1,94 @@
...
 
 ## Conclusion
 
-XXXTBDXXX
+VSTS enables you to start small, take your solution from idea to release, scale-up as needed, and leave the support and maintenance of services to the experts.
+
+![Choose the recommended option](_img/understand-options-tfs-vsts-environments/understand-options-tfs-vsts-environments-select.png)
 
-## Q&A
+It's only when you want to manage servers and services yourself, that the on-premises or virtualised TFS single, dual, and multi server scenarios (7-9) are recommended.
```
Comment (kenmuse) [inline src/Planning/understand-options-tfs-vsts-environments.md]: Comma not needed after *manage servers and services yourself*.

File: src/Planning/understand-options-tfs-vsts-environments.md [hunk 4]
```diff
@@ -1,33 +1,94 @@
...
+|Dual Servers (scale up)|476-730|2200-3600|
+|Multi Servers (scale out)|730+|3600+|
+
+>
+> [!TIP]
+>
+> The dual and multi server options allow you to scale your TFS environment as your organization or number of power users grow.
+>
+
+If you have remote sites, sharing code with your main TFS server site, the [Team Foundation Server Proxy](https://docs.microsoft.com/en-us/tfs/server/install/install-proxy-setup-remote) helps to cache recently used version control files, improve remote user experience, and reduce network load.
```
Comment (kenmuse) [inline src/Planning/understand-options-tfs-vsts-environments.md]: Don't think comma is required after *remote sites*

File: src/Planning/understand-options-tfs-vsts-environments.md (hunk 1 above)
Comment (wpschaub) [reply src/Planning/understand-options-tfs-vsts-environments.md]: Good point. Moved.

File: src/Planning/understand-options-tfs-vsts-environments.md (hunk 3 above)
Comment (wpschaub) [reply src/Planning/understand-options-tfs-vsts-environments.md]: Removed

File: src/Planning/understand-options-tfs-vsts-environments.md (hunk 4 above)
Comment (wpschaub) [reply src/Planning/understand-options-tfs-vsts-environments.md]: Will defer to copy editing.

File: src/Planning/understand-options-tfs-vsts-environments.md (hunk 2 above)
Comment (wpschaub) [reply src/Planning/understand-options-tfs-vsts-environments.md]: Yup. Done.

**Pull request merged**